GVFS/GVFS.Platform.POSIX/POSIXPlatform.cs

```diff
@@ -38,9 +38,9 @@ public override void ConfigureVisualStudio(string gitBinPath, ITracer tracer)
         {
         }
 
-        public override bool TryGetGVFSHooksPathAndVersion(out string hooksPaths, out string hooksVersion, out string error)
+        public override bool TryGetGVFSHooksVersion(out string hooksVersion, out string error)
         {
-            hooksPaths = string.Empty;
+            string hooksPaths = string.Empty;

```

Comment:
Let's condense `hooksPath` and `binPath`, since I think they're effectively the same? They're also unused in this particular method, but I think the TODO tracks that work.